src/Clue/Redis/Protocol/Parser/ResponseParser.php

```diff
@@ -96,7 +96,7 @@ private function readResponse() {
         if ($reply === null) {
             return null;
         }
-        switch (substr($reply, 0, 1)) {
+        switch ($reply[0]) {
```

Comment:
We may want to support empty strings here, so this now likely requires an additional `isset()`?